### PR TITLE
Circleci fix

### PR DESCRIPTION
**Pull request by aa-ahmed-aa**
## Description
CircleCI config.yml changes and make sqlite run from the storage/database.sqlite
#3

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -49,10 +49,11 @@ jobs:
             - node_modules
```
Comment (SalemAttia) [inline .circleci/config.yml:49]: what this for ?

File: .circleci/config.yml [hunk 2]
```diff
@@ -49,10 +49,11 @@ jobs:
...
 
       # prepare the database
+      - run: cp .env.testing .env
```
Comment (SalemAttia) [inline .circleci/config.yml:52]: it's testing env why we add it ?

File: .circleci/config.yml [hunk 3]
```diff
@@ -49,10 +49,11 @@ jobs:
...
-      - run: php artisan migrate --env=testing --database=sqlite_testing --force
+      - run: php artisan migrate
 
       # run tests with phpunit or codecept
-      #- run: ./vendor/bin/phpunit
-      - run: ./vendor/bin/codecept build
-      - run: ./vendor/bin/codecept run
\ No newline at end of file
+      - run: ./vendor/bin/phpunit
+      # - run: ./vendor/bin/codecept build
```
Comment (SalemAttia) [inline .circleci/config.yml:58]: what this for

File: config/database.php [hunk 4]
```diff
@@ -48,7 +48,7 @@
 
         'sqlite' => [
             'driver' => 'sqlite',
-            'database' => env('DB_DATABASE', database_path('database.sqlite')),
+            'database' =>  storage_path() . DIRECTORY_SEPARATOR . env('DB_DATABASE','database') . '.sqlite',
```
Comment (SalemAttia) [inline config/database.php:51]: why you did that

File: tests/ExampleTest.php [hunk 5]
```diff
@@ -13,7 +13,6 @@ class ExampleTest extends TestCase
      */
     public function testBasicExample()
     {
-        $this->visit('/')
```
Comment (SalemAttia) [inline tests/ExampleTest.php:16]: it's configuration for circleCi plz remove this test as it was before on master

**Review by SalemAttia — CHANGES_REQUESTED** [review]
@mustafah15  can you review this plz

File: .circleci/config.yml (hunk 1 above)
Comment (aa-ahmed-aa) [reply .circleci/config.yml:49]: i don't know this is template from circleci

File: .circleci/config.yml (hunk 2 above)
Comment (aa-ahmed-aa) [reply .circleci/config.yml:52]: substitute the normal production env and circleci env every one for it's default configuration

File: .circleci/config.yml (hunk 3 above)
Comment (aa-ahmed-aa) [reply .circleci/config.yml:58]: this line is commented, 
B.T.W This is codecept test runner we use phpunit not this package

File: config/database.php (hunk 4 above)
Comment (aa-ahmed-aa) [reply config/database.php:51]: loading the sqlite from the storage folder not database cause database is tracked by git

File: tests/ExampleTest.php (hunk 5 above)
Comment (aa-ahmed-aa) [reply tests/ExampleTest.php:16]: Thank you Mr. salem for reviewing nothing

Comment (SalemAttia) [reply tests/ExampleTest.php:16]: i mean delete the whole test case it's not good to have test case not doing any thing !!!

**Review by SalemAttia — APPROVED** [review]

**Pull request merged**